src/test/groovy/org/prebid/server/functional/model/response/cookiesync/UsersyncInfo.groovy

```diff
@@ -11,12 +11,9 @@ class UsersyncInfo {
     Boolean supportCORS
 
     enum Type {
-
-        REDIRECT, IFRAME
-
-        @JsonValue
-        String getValue() {
-            name().toLowerCase()
-        }
+        @JsonProperty("iframe")
```

Comment:
I prefer an old way of setting one logic for all values with `JsonValue` instead of providing a name for each property manually.